README.md

```diff
@@ -150,6 +150,7 @@ The `--fix` option on the command line automatically fixes problems reported by
 
 |    | Rule ID | Description |
 |:---|:--------|:------------|
+|  | [no-pause-test](./docs/rules/no-pause-test.md) | Disallow usage of pauseTest() helper in tests |
```

Comment:
Would the rule name be better as `no-test-pause-test`?

Reply:
I'm fine with `no-pause-test` since it's already clear that it's about tests.

Also there are a few other testing rules that do not start with `no-test` (such as `no-restricted-resolver-tests`).